package.json

```diff
@@ -365,6 +367,7 @@
     "date-fns": "^2.15.0",
     "fuse.js": "6.6.2",
     "jszip": "^3.2.2",
+    "jszip-unzip": "^3.4.0-0",
```

Comment:
should this go in devDependencies? and also pin the exact version please

Reply:
actually looks like https://www.npmjs.com/package/jszip-unzip is not maintained. maybe you can just use a shell command instead of this dependency? https://stackabuse.com/executing-shell-commands-with-node-js/

Reply:
Dependency removed.